### DISCUSSION TITLE
When will the community version be released with latest conductor 3.13.2

### DISCUSSION THREAD

**Discussion by sks1995** [Netflix/conductor-community#145] [Q&A]
When will the community version be released with latest conductor 3.13.2

**Comment by shekarpcs** [comment]
Even we are waiting for it . When will it be available.

**Comment by v1r3n** [comment]
HI @shekarpcs its released.  Please let us know if you see any issues.

**Comment by shekarpcs** [comment]
Thanks for the quick build. It helps a lot .
